### DISCUSSION TITLE
How to read from browser data? (cookie/localstorage)

### DISCUSSION THREAD

**Discussion by wooparadog** [Budibase/budibase#8305] [General]
Hi,

Is there a way to get localstorage data and use it on rest APIs?

background:

1. Our company's API auth is based on a specific token stored in browser's localstorage.
2. I'm trying to integrate our company's API with buildbase, but failed to get the token that's stored in localstorage and pass it to rest api

Thanks in advance.

**Comment by andz-bb** [comment]
hey @wooparadog, unfortunately I don't think this is possible. Local storage is generally only going to be accessible for the current domain so as far as I'm aware I don't think this can be done. Additionally the bindings for REST APIs only currently support Handlebars bindings, so localStorage won't be accessible from there the way it would be when using JS bindings.

**Comment by wooparadog** [comment]
then is cookie limited as well?...feel weird if user cookie is stripped as well
